site/content/blog/2022-01-01-whats-new-in-svelte-january-2022.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "What's new in Svelte: January 2022"
+description: "Faster builds with SvelteKit and a much anticipated REPL feature"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Happy new year, Svelte Community! Lots to share this month across Svelte, SvelteKit, Language Tools and the Showcase. Thanks to everyone who made 2021 a great year to use Svelte. Looking forward to the next one 🚀
+
+## What's new in `svelte/*`
+- [Svelte 3.44.3](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md#3443) is out with a few bug fixes in the binding and loop code
+- Svelte Language Tools has introduced support for the then/catch shorthands from Svelte 3.41 and TypeScript's "go to" functionality ([105.8.0 and later](https://github.com/sveltejs/language-tools/releases/tag/extensions-105.8.0))
+- After the [update to Vite 2.7](https://github.com/sveltejs/kit/pull/3018), SvelteKit users are [reporting significant performance improvements](https://www.reddit.com/r/sveltejs/comments/rljhfc/sveltekit_massive_compiler_improvement_by/)
```

Comment:
It'd also be worth mentioning that loading third-parties libraries in SSR is greatly improved in Vite 2.7